ffcompare.py

```diff
@@ -68,6 +68,16 @@
 
 
 def writeUpdatedMol(Mol, fname):
+    """
+    
+    Parameters
+    ----------
+    Mol: an OEChem molecule
+    
+    Returns
+    ----------
+    Output mol2 file
```

Comment:
Specify in the returns section: `fname: str - name of the output mol2 file` to be clear what the data type being returned is.